fbw-a32nx/src/systems/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -294,7 +303,9 @@ enum TargetAltitudeColor {
 export class AltitudeIndicatorOfftape extends DisplayComponent<AltitudeIndicatorOfftapeProps> {
   private abnormal = FSComponent.createRef<SVGGElement>();
 
-  private tcasFailed = FSComponent.createRef<SVGGElement>();
+  private altFlagVisible = Subject.create(false);
```

Comment:
```suggestion
  private readonly altFlagVisible = Subject.create(false);
```
It's generally a good idea to make objects readonly where possible, just to make sure nobody can assign them and create really confusing bugs.